### ISSUE TITLE
Only page-load transactions have longtask metrics in `transaction.experience.longtask`

### ISSUE THREAD

**Issue by clyfish** [elastic/apm-agent-rum-js#938]
There are longtask spans in other transactions(user-interaction, http-request), but only page-load transactions have longtask metrics in `transaction.experience.longtask`.

**Comment by vigneshshanmugam** [comment]
@clyfish Good idea, totally makes sense. We will add it to our next roadmap item.

**Comment by litwin90** [comment]
@clyfish Good idea
